src/main/kotlin/racingcar/domain/car/Cars.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain.car
+
+import racingcar.domain.Name
+import racingcar.domain.numberpicker.NumberPicker
+
+class Cars(val cars: List<Car>) {
+    fun play(numberPicker: NumberPicker): Cars {
+        return Cars(cars.map { car -> car.moveOverCondition(numberPicker.getNumber()) })
+    }
+
+    fun getWinners(): Winners {
+        val maxPosition = cars.maxOf { it.position }
+        return Winners(cars.filter { it.position == maxPosition })
+    }
+
+    companion object {
+        fun from(names: List<Name>): Cars {
+            return Cars(names.map { Car(it) })
```

Comment:
```suggestion
            return Cars(names.map(::Car))
```

How about using a function reference in this situation? 

I prefer using function references when I can, check out this article to learn more about it!

https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845
> To sum up, the difference between lambda and method reference is that when we use method reference, the instance of variable, the method of which we use, is fixed at the moment the reference is created, not when it is called, unlike what happens when lambda is used.